packages/blockchain-link/src/workers/solana/tokenUtils.ts

```diff
@@ -0,0 +1,3 @@
+import { TokenDetailByMint } from '@trezor/blockchain-link-types/lib/solana';
+
+export const fetchSolanaTokenList = async (): Promise<TokenDetailByMint> => Promise.resolve({});
```

Comment:
Since we introduced this in the first commit, I would leave a PR comment here saying 
`The two alternatives of implementing this function are shown in the next two commits` or something like that